dashboard/app/models/sections/section_teacher.rb

```diff
@@ -0,0 +1,35 @@
+# == Schema Information
+#
+# Table name: section_teachers
+#
+#  id            :bigint           not null, primary key
+#  teacher_id    :integer          not null
+#  section_id    :integer          not null
+#  invited_by_id :integer
+#  deleted_at    :datetime
+#  status        :integer          not null
+#  created_at    :datetime         not null
+#  updated_at    :datetime         not null
+#
+# Indexes
+#
+#  index_section_teachers_on_deleted_at                 (deleted_at)
+#  index_section_teachers_on_invited_by_id              (invited_by_id)
+#  index_section_teachers_on_section_id                 (section_id)
+#  index_section_teachers_on_teacher_id                 (teacher_id)
+#  index_section_teachers_on_teacher_id_and_section_id  (teacher_id,section_id) UNIQUE
+#
+class SectionTeacher < ApplicationRecord
```

Comment:
One thought: we've sort of slowly been moving the term "teacher" and "student" to be "instructor" and "participant" because PL sections have participants who are teachers. If it wouldn't make things too confusing, it might be good to use those terms here.

Reply:
This is a presentation by Dani (who implemented a lot of this work) on why we changed those terms: https://docs.google.com/presentation/d/1EeVOHGLQGS3Q79tT2JP1kOBYHENnOsExH8NE4FbOMfI/edit#slide=id.p

Reply:
Done.